Rakefile

```diff
@@ -30,5 +30,35 @@ Rake::TestTask.new(:test) do |t|
   t.verbose = false
 end
 
+namespace :deploy do
+  desc "Update the sandbox app on Github"
+  task :update do
+    sandbox = "godmin-sandbox"
+    sandbox_dir = "/tmp/#{sandbox}-#{Time.now.strftime("%Y%m%d")}"
+    puts "Working in #{sandbox_dir}"
+    template_file = File.expand_path("../template.rb", __FILE__)
+    system("rm -rf #{sandbox_dir}")
+    system("mkdir #{sandbox_dir}")
+    system("cd #{sandbox_dir} && git clone git@github.com:varvet/#{sandbox}.git")
+    system("cd #{sandbox_dir}/#{sandbox} && rm -rf *")
+    system("cd #{sandbox_dir} && rails new #{sandbox} -m #{template_file} --without-engine")
```

Comment:
Can't you use `--database=postgres` here instead of adding things to the template?